src/compose/utils.ts

```diff
@@ -190,8 +190,13 @@ function getNanoseconds(timeStr: string): number {
 
 export function composeHealthcheckToServiceHealthcheck(
 	healthcheck: ComposeHealthcheck | null | undefined,
-): ServiceHealthcheck {
-	if (healthcheck == null || healthcheck.disable) {
+): ServiceHealthcheck | { } {
```

Comment:
nit: Should there be no space in the `{ }`?

Reply:
Good catch